SECURITY.md

```diff
@@ -1,15 +1,5 @@
-Security Policy
----------------
+# Security Policy
+Intel is committed to rapidly addressing security vulnerabilities affecting our customers and providing clear guidance on the solution, impact, severity and mitigation.
 
-Intel is committed to rapidly addressing security vulnerabilities
-affecting our customers and providing clear guidance on the solution,
-impact, severity and mitigation.
-
-
-Reporting a Vulnerability
--------------------------
-
-Please report any security vulnerabilities in this project utilizing the
-guidelines [here].
-
-[here]: https://www.intel.com/content/www/us/en/security-center/vulnerability-handling-guidelines.html
+## Reporting a Vulnerability
+Please report any security vulnerabilities in this project utilizing the guidelines [here](https://www.intel.com/content/www/us/en/security-center/vulnerability-handling-guidelines.html).
```

Comment:
Since URL cannot be truncated, please put them at the very end and reference them from within the text.